configs.go

```diff
@@ -217,6 +223,67 @@ func (config MessageConfig) method() string {
 	return "sendMessage"
 }
 
+// PollConfig contains information about a SendPoll request.
+type PollConfig struct {
+	BaseChat
+	Question              string
+	Options               []string
+	Anonymous             bool
```

Comment:
Let's change this to `IsAnonymous` to keep it consistent with the other fields prefixed with `is_` in the API.